graphql/ui_plugin_build_baron.go

```diff
@@ -52,11 +52,24 @@ func BbFileTicket(context context.Context, taskId string, execution int) (bool,
 	settings := env.Settings()
 	queue := env.RemoteQueue()
 	bbProject, ok := BbGetProject(settings, t.Project)
+	var webHook evergreen.WebHook
+	flags, err := evergreen.GetServiceFlags()
+	if err != nil {
+		return false, errors.Wrap(err, "error retrieving admin settings")
+	}
+	if flags.PluginAdminPageDisabled {
+		projectRef, err := model.FindOneProjectRef(t.Project)
```

Comment:
need to check if projectRef is nil. Also, we'll want to use `FindMergedProjectRef` so that we include the logic of the repo settings. (I think we aren't doing this right in a lot of places; I'll probably audit existing usages to make this more clear)